lib/spritesheet.dart

```diff
@@ -1,58 +0,0 @@
-import 'sprite.dart';
```

Comment:
Why remove this class?

This class is a great utility to extract sprites and animations from a single spritesheet image without having to manually calculating the coordinates.